README.md

```diff
@@ -36,7 +36,7 @@ to see which Kubernetes and Openshift versions the Operator is compatible with
 To work with MongoDB resource this Operator requires [Ops Manager](https://docs.opsmanager.mongodb.com/current/) (Ops Manager can 
 be installed into the same Kubernetes cluster by the Operator or installed outside of the cluster manually) 
 or [Cloud Manager](https://cloud.mongodb.com/user#/cloud/login).
-> If this is your first time trying the Operator, Cloud Manager is easier to get started
+> If this is your first time trying the Operator, Cloud Manager is easier to get started.  Use or create a Cloud Manager based organization.  Atlas-based organizations will not work. 
```

Comment:
our official terminology is "Cloud Manager Organization / Atlas Organization" - could you drop the "-based" suffix here please?